### DISCUSSION TITLE
INFO Cannot connect to host 127.0.0.1:8444 ssl:

### DISCUSSION THREAD

**Discussion by automation13** [Chia-Network/chia-blockchain#6855] [Support]
Hi guys, how to resolve this issue? 
`farmer farmer_server : INFO Cannot connect to host 127.0.0.1:8444 ssl:`

So what I'm doing is:
Install Chia
Open chia typed my 24 words
Waited until sync starts
Close chia app
Copy/paste my walled DB and blockchain db to .chia/mainnet folder
Open chia and nothing happens. 

I tried pretty much everything...

![image](https://user-images.githubusercontent.com/11642218/122600800-4f2f9400-d03e-11eb-89dc-20e7c60c0de8.png)

Trying to figure out how to fix it for 2 days now...

Win 10
chia 1.1.7
yes, port 8444 open

**Comment by loppefaaret** [comment]
try this:
WALLET DATABASE RESET

Close down the Chia app, and open up a PowerShell window, and execute the below command:
```
mv ~\.chia\mainnet\wallet\db\block* ~\Desktop\
```
Then try opening up the Chia app again
if it worked, you can remove the files we put on the Desktop!

**Comment by BrandtH22** [comment]
@automation13 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
